src/Client/Client.cs

```diff
@@ -0,0 +1,123 @@
+using Newtonsoft.Json;
+using System;
+using System.Buffers;
+using System.IO.Pipelines;
+using System.Net.Sockets;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace TcpEcho
+{
+    public class BlockchainTransactionGet {
+        [JsonProperty("id")]
+        public int Id { get; set; }
+        [JsonProperty("result")]
+        public string Result { get; set; }
+    }
+
+    public class Client
+    {
+        private const int minimumBufferSize = 1024;
+        private string endpoint = "";
+        private int port = 0;
+
+        public Client(string _endpoint, int _port) {
+            endpoint = _endpoint;
+            port = _port;
+        }
+
+        public async Task<string> BlockchainTransactionGet(string transactionId) {
```

Comment:
rather place this method in a new class called StratumClient which inherits from Client

Reply:
Done